ingest/mongo_connection.py

```diff
@@ -52,13 +52,12 @@ def graceful_auto_reconnect(mongo_op_func):
     import random
     import math
 
-    MAX_ATTEMPTS = 5
     # Adopted from https://stackoverflow.com/questions/46939285
 
     def retry(attempt_num):
-        if attempt_num < MAX_ATTEMPTS - 1:
+        if attempt_num < MongoConnection.MAX_AUTO_RECONNECT_ATTEMPTS - 1:
             exp_backoff = pow(2, attempt_num)
```

Comment:
AIUI, an issue this PR addresses is that the backoff period was too brief, not that retries were too synchronized.  

So given no signs of [thundering herds](https://en.wikipedia.org/wiki/Thundering_herd_problem) per se, I suspect we could more simply -- and maybe more effectively -- mitigate failed large ingests by increasing the underlying backoff, while leaving jitter unchanged.  The change's wider jitter does increase backoff, which indeed ought to help, just more subtly and indirectly.

Below is a trivial way we could implement that.  The cost to test it would be non-trivial, though, so the refinement suggested below doesn't strike me as worthwhile _unless_ we see future similar failures of large ingests.

```suggestion
            exp_backoff = pow(2, attempt_num + 1)
```